teamcity/build_feature.go

```diff
@@ -131,6 +161,28 @@ func (s *BuildFeatureService) Delete(id string) error {
 	return nil
 }
 
+// DeleteAll removes all build features of a build configuration
+func (s *BuildFeatureService) DeleteAll() error {
+	emptyStruct := struct{}{}
+	req, err := s.base.New().Put("").BodyJSON(emptyStruct).Request()
+
+	if err != nil {
+		return err
+	}
+
+	resp, err := s.httpClient.Do(req)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("Error when removing build features, statusCode: %d", resp.StatusCode)
```

Comment:
Can we include more detail in the error, such as the body as a string? The `Delete` function above demonstrates how to do this.

Also (minor nitpick), can the language be changed to `"Error '%d' when deleting all build features: %s"`, to be consistent with `Delete`'s language?